packages/common/src/client/appeals-api-client.js

```diff
@@ -50,6 +50,25 @@ class AppealsApiClient {
 		return response.json();
 	}
 
+	/**
+	 * @param {string} ref
+	 * @returns {Promise<boolean>}
+	 */
+	async appealCaseRefExists(ref) {
+		const endpoint = '/api/v2/appeal-cases/' + ref;
```

Comment:
just a thought for another time, and probably a yagni, maybe the api version could be an optional constructor param for this class